helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/AvscWriter.java

```diff
@@ -51,6 +97,7 @@ public String toAvsc(Schema schema) {
             gen.flush();
             return writer.toString();
         } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
i assume this is left over?

Reply:
yes. Will remove. Thanks for pointing out